### DISCUSSION TITLE
Is this an official GitHub SSH key fingerprint?

### DISCUSSION THREAD

**Discussion by WPum** [community/community#37174] [Other Feature Feedback, Questions, & Ideas] [answered]
`ssh-rsa 2048 SHA256:rEmlJenVMSL5GVemSY0Gk8WGw6B4ege4J85M+vup8R0`

I received this key from the GitHub SSH server when I tried to push some commits to GitHub, but I couldn't find any information about this fingerprint.

**Comment by elli0t43** [comment]
> `ssh-rsa 2048 SHA256:rEmlJenVMSL5GVemSY0Gk8WGw6B4ege4J85M+vup8R0`

It is. The end part of the fingerprint is your main hash.

**Reply by airtower-luna** [reply]
That is dangerously wrong, the fingerprint is *not* one of those published by GitHub (link below). :warning:

**Reply by elli0t43** [reply]
My bad I didn't read the github SSH server.

**Comment by airtower-luna** [comment, marked as answer]
It is not, do not connect to that server. GitHub publishes their list of fingerprints here: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/githubs-ssh-key-fingerprints

The one you posted is not one of those, so something is wrong. :warning:
